src/index.ts

```diff
@@ -104,6 +104,7 @@ function deriveKeypair(seed, options) {
   const keypair = method.deriveKeypair(decoded.bytes, options)
   const messageToVerify = hash('This test message should verify.')
   const signature = method.sign(messageToVerify, keypair.privateKey)
+  /* istanbul ignore if */
   if (method.verify(messageToVerify, signature, keypair.publicKey) !== true) {
     throw new Error('derived keypair did not generate verifiable signature')
```

Comment:
Is it possible to write a test that throws this error?

Reply:
I don't think so - it is a "just in case" verification. (Historically there was a bug in Safari that could cause this error to be thrown.)